vmdk_plugin/utils/test_util/VolumeCreationTestUtil.go

```diff
@@ -49,3 +53,130 @@ func DeleteVolume(name string, ip string) ([]byte, error) {
 	return exec.Command("/usr/bin/ssh", append(SSH_OPTS, "root@"+remoteHostIP, cmd)...).CombinedOutput()
 }
+
+// returns disk format of volume using Admin cli
+func GetDiskFormat_AdminCli(volName string, hostname string) string {
+	cmd := "/usr/lib/vmware/vmdkops/bin/vmdkops_admin.py ls -c volume,disk-format | grep " + volName
+	op := ExecCmd(hostname, cmd)
+	result := strings.Fields(op)
+	diskFormat := result[1]
+	fmt.Println("disk format", diskFormat)
+	return string(diskFormat)
+}
+
+// returns disk format of volume using docker cli
+func GetDiskFormat_DockerCli(volName string, hostname string) string {
+	cmd := "docker volume inspect " + volName
+	op := ExecCmd(hostname, cmd)
+	volProps := strings.Fields(op)
+	i := 0
+	for range volProps {
+		if volProps[i] == "\"diskformat\":" {
+			i = i + 1
+			break
+		}
+		i++
+	}
+	fmt.Println("disk format is  - ", volProps[i])
+	df := strings.Trim(volProps[i], "\"")
+	dskFormat := strings.Trim(df, ",")
+	result := strings.Trim(dskFormat, "\"")
+	return result
+}
+
+// returns attached to vm field of volume using docker cli
+func GetVmAttachedToVol_DockerCli(volName string, hostname string) string {
+	cmd := "docker volume inspect " + volName
+	op := ExecCmd(hostname, cmd)
+	volProps := strings.Fields(op)
+	i := 0
+	for range volProps {
+		if volProps[i] == "\"attached" && volProps[i+1] == "to" && volProps[i+2] == "VM\":" {
+			i = i + 3
+			break
+		}
+		i++
+	}
+	vm := strings.Trim(volProps[i], "\"")
+	vmName := strings.Trim(vm, ",")
+	result := strings.Trim(vmName, "\"")
+	return result
+}
+
+// returns attached to vm field of volume using admin cli
+func GetVmAttachedToVol_AdminCli(volName string, hostname string) string {
+	cmd := "/usr/lib/vmware/vmdkops/bin/vmdkops_admin.py ls -c volume,attached-to | grep " + volName
+	op := ExecCmd(hostname, cmd)
+	volProps := strings.Fields(op)
+	vmName := volProps[1]
+	return string(vmName)
+}
+
+// returns capacity of volume using docker cli
+func GetVolSizeUsingDockerCli(volName string, hostname string) string {
+	cmd := "docker volume inspect " + volName
+	op := ExecCmd(hostname, cmd)
+	volProps := strings.Fields(op)
+	i := 0
+	for range volProps {
+		if volProps[i] == "\"size\":" {
+			i = i + 1
+			break
+		}
+		i++
+	}
+	sz := strings.Trim(volProps[i], "\"")
+	size := strings.Trim(sz, ",")
+	result := strings.Trim(size, "\"")
+	return result
+}
+
+// returns capacity of volume using admin cli
+func GetVolSizeByAdminCli(volName string, hostname string) string {
+	cmd := "/usr/lib/vmware/vmdkops/bin/vmdkops_admin.py ls -c volume,capacity | grep " + volName
+	op := ExecCmd(hostname, cmd)
+	volProp := strings.Fields(op)
+	volSize := volProp[1]
+	return string(volSize)
+}
+
+// returns docker version
+func GetDockerVersion(hostname string) string {
+	cmd := "docker -v"
+	op := ExecCmd(hostname, cmd)
+	return op
+}
+
+// method returns unique name.
+// can be used to create a volume with unique name
+func GenerateTestVolName() string {
```

Comment:
this can be replaced with https://github.com/vmware/photon/wiki/Frequently-Asked-Questions#how-to-install-new-packages